.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -1,4 +1,16 @@
 # zexi 0.2.0
+cross_SnsPublishSubscription:
+  title: Creating an &SNSlong; Publish and Subscription Web Application that Translates Messages
```

Comment:
Create an &SNS; publish and subscription web application that translates messages
* Imperative verb for titles.
* Use short entity in titles: &SNS;
* Sentence case for titles.

Reply:
fixed